include/zephyr/pm/state.h

```diff
@@ -156,6 +156,15 @@ struct pm_state_info {
 	uint32_t exit_latency_us;
 };
 
+/**
+ * @enum pm_state_direction Entry/exit indicator for state transitions
+ */
+enum pm_state_direction {
+	PM_STATE_ENTRY,
+	PM_STATE_EXIT,
+	PM_STATE_ENTRY_AND_EXIT,
```

Comment:
I think this should just be a bit field, that you can OR if you need both.

Reply:
That'd indeed be more elegant. I'll try that out, thanks.

Reply:
yep, that is better indeed.

Reply:
Done.